src/content-tags-drawer/TagBubble.jsx

```diff
@@ -8,35 +8,45 @@ import PropTypes from 'prop-types';
 import TagOutlineIcon from './TagOutlineIcon';
 
 const TagBubble = ({
-  value, subTagsCount, implicit, level,
+  value, implicit, level, lineage, removeTagHandler,
 }) => {
   const className = `tag-bubble mb-2 ${implicit ? 'implicit' : ''}`;
   const tagIcon = () => (implicit ? <TagOutlineIcon className="implicit-tag-icon" /> : <Tag />);
```

Comment:
I don't think you need this function to add the `.implicit-tag-icon` class. I found it works just fine with `iconBefore={!implicit ? Tag : TagOutlineIcon}`

**but**, there is a bug in the `TagOutlineIcon.jsx` file:

You need to remove the line `<path d="M0 0h24v24H0V0z" fill="none" />` which is basically rendering an invisible rectangle over top of the icon. Sometimes the SCSS makes that rectangle not so invisible:

![Screenshot 2023-11-22 at 3 29 49 PM](https://github.com/openedx/frontend-app-course-authoring/assets/945577/dd17365a-ba3e-4553-aa50-503bbc82fd2b)

But remove that rectangle and it works:
![Screenshot 2023-11-22 at 3 30 15 PM](https://github.com/openedx/frontend-app-course-authoring/assets/945577/aae5578b-92e0-4301-8ed6-f82cf59643fc)

Reply:
Thanks for catching this! Updated it.